src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,153 @@
   }
 }
+
+@mixin data-table-v2-core--x {
+  //----------------------------------------------------------------------------
+  // Container
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-01; // allow space for focus styles
+  }
+
+  //----------------------------------------------------------------------------
+  // Table title text
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //----------------------------------------------------------------------------
+  // Data table
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 thead {
+    @include type-style('heading-01');
+    background-color: $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 tbody {
+    @include type-style('body-short-01');
+    background-color: $ui-01;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tr {
+    border: none;
+    height: $layout-04;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover {
+    background: $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover td {
+    color: $text-01;
+    background: $hover-field;
+    border-bottom: 1px solid $hover-field;
+    border-top: 1px solid $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 th,
+  .#{$prefix}--data-table-v2 td {
+    padding-left: $spacing-04;
+    padding-right: $spacing-04;
+    vertical-align: top;
+    text-align: left;
+  }
+
+  .#{$prefix}--data-table-v2 th {
+    color: $text-01;
+    background-color: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+    padding-left: $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2 .#{$prefix}--table-header-label {
+    display: block;
+    padding: rem(14px) 0;
+  }
+
+  .#{$prefix}--data-table-v2 td {
+    background: $ui-01;
+    color: $text-02;
+    border-top: 1px solid $ui-01;
+    border-bottom: 1px solid $ui-03;
+    padding: rem(14px) $spacing-04;
+  }
+
+  .#{$prefix}--data-table-v2 td:first-of-type {
+    padding-left: $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2 td:last-of-type {
+    padding-right: $spacing-05;
+  }
+
+  // specific padding for overflow menu columns
+  .#{$prefix}--data-table-v2 .#{$prefix}--table-column-menu,
```

Comment:
Could we target `bx--overflow-menu` directly instead of adding a new selector? Or do we need to offset padding at the top (`td`) level?

Reply:
This is padding for the column that the overflow menu is in, not the menu itself. If the column has a menu, it has different padding than the other columns. See Tyler's comment about the overflow column here: https://github.com/IBM/carbon-components/pull/1642#issuecomment-472155022. All other `:last-of-type` columns have 16px right padding.

Reply:
That's why I added a class to it, but also because for the `th's` if it's a sortable table, the header shouldn't have a background color change the way other `th`s do, since it can't be sorted. The class is used as a selector to specify that (though it's not included in this PR).

Reply:
Is the intent that each icon would be in its own column? For this case:

![image](https://user-images.githubusercontent.com/3901764/54724056-91a05880-4b37-11e9-88bd-932a560f78c3.png)

Or would they all be wrapped in the same column? If the latter, might help to add a new class like `--table-row-actions` that we could use to generalize this for any action instead of tying it just to a menu 👍